techniques/ios/MASTG-TECH-0084.md

```diff
@@ -55,55 +55,55 @@ Note: On iOS 12 and higher, use the following procedure to sign the debugserver
 
 2) Connect to the device via SSH and create the file, named entitlements.xml, with the following content:
 
-    ```xml
```

Comment:
See https://github.com/DavidAnson/markdownlint/issues/125

Reply:
Was this reporting MD046: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md046---code-block-style